internal/dao/sqlx/account_dao.go

```diff
@@ -76,7 +74,7 @@ func (di *accountDaoSqlx) Create(ctx context.Context, account *models.Account) e
 	query := createAccount
 	stmt := di.create
 
-	err := stmt.GetContext(ctx, account, account.AccountNumber, account.OrgID)
+	err := stmt.GetContext(ctx, account, account.AccountNumber.String, account.OrgID)
```

Comment:
Should we validate for x.Valid?

Reply:
Not here, NULL is valid here.

Reply:
But the problem is, you need to pass the AccountNumber struct, not string. The sql/sqlx library will take care of storing either NULL or value.